pandas/core/window.py

```diff
@@ -899,11 +899,45 @@ def skew(self, **kwargs):
         return self._apply('roll_skew', 'skew',
                            check_minp=_require_min_periods(3), **kwargs)
 
-    _shared_docs['kurt'] = """Unbiased %(name)s kurtosis"""
+    _shared_docs['kurt'] = dedent("""Calculate unbiased %(name)s kurtosis.
 
-    def kurt(self, **kwargs):
+    This function uses Fisher's definition of kurtosis (kurtosis of normal
+    == 0.0) without bias.
+
+    Returns
+    -------
+    same type as input
+
+    See Also
+    --------
+    scipy.stats.kurtosis
+    pandas.DataFrame.kurtosis
+    pandas.Series.kurtosis
```

Comment:
I think it could make sense to add `.skew` too as statistically they are related, and usually appear together in literature. And technically speaking the mean and the variance could also belong here, as they are the first and second moments of a probability density (while skewness and kurtosis are the third and forth).

Also, it's probably worth to add `pandas.Series.rolling`. And not sure if in the "See Also" or somewhere else, but I think it would be very useful for the users to have a link from these functions to https://pandas.pydata.org/pandas-docs/stable/computation.html#window-functions

Reply:
Yea I was thinking the same thing about `pandas.Series.rolling` and `pandas.Series.expanding`. Right now there is the `_doc_template` contained within the module that gives a generic `See Also` section, but my first attempt to override that didn't work out. 

I'll look a little closer to see if there's a scalable way to have all of the functions here reference `pandas.Series.rolling` and the Series / DataFrame methods at a minimum, while allowing for additional "see also" to be specified per-method